doc/user_guide.md

```diff
@@ -123,7 +116,8 @@ Note: We've found that some versions of TLS in hardware do not have all of these
 
 # Xively Client Standard Features
 
-The Xively Client Agent for C was designed with embedded systems in mind. That is, it is a low footprint, low memory usage, communications solution that can run on a single thread with a single socket connection on multiple operating systems. But the client itself also offers a suite of subsystems that go beyond a standard communications API.
+The Xively Client Agent for C was designed with embedded systems in mind. It has a low footprint, minimal memory usage, and a scheduling system to make your application's use of MQTT as easy as possible.  It can run on a single thread with a single socket connection on multiple operating systems.  The following are details about all of these subsystems.
```

Comment:
"The following are details about all of these subsystems."
The following details are about ... ?